MaskRCNN_no_batch_convergence/train.py

```diff
@@ -184,8 +225,10 @@ def __init__(self, fp16):
     def inputs(self):
 
         ######################################################################################################
-        if BATCH_DATA_PIPELINE:
+        if BATCH_DATA_PIPELINE or BATCH_EVAL:
         ######################################################################################################
+            assert(BATCH_DATA_PIPELINE and BATCH_EVAL)
```

Comment:
So BATCH_DATA_PIPELINE and BATCH_EVAL always have to either both be False or both be True? Should we just combine them into a single flag in that case?

Reply:
BATCH_DATA_PIPELINE and BATCH_EVAL combined into BATCH_DATA_PIPELINE_EVAL.